### PR TITLE
[17.09] Fix auth with ldaps://

### PR DESCRIPTION
**Pull request by abretaud**
Update the python-ldap requirement, the new version allows connecting to ldaps:// now
See galaxyproject/starforge#146 and #3178

### CODE REVIEW

**Comment by nsoranzo** [comment]
@abretaud I thought you wanted to target release_17.09?

**Comment by abretaud** [comment]
Oh yes, I forgot about it, I just changed the base branch (and rebased)

**Review by nsoranzo — APPROVED** [review]

**Comment by abretaud** [comment]
I don't remember how it works: do I need to make a PR against dev branch too?

**Comment by nsoranzo** [comment]
@abretaud No, don't worry, it will be merged forward by a committer after the PR is merged.

**Comment by jmchilton** [comment]
Awesome thanks all!

**Pull request merged**
